Code/GraphMol/Aromaticity.cpp

```diff
@@ -417,13 +418,21 @@ void applyHuckelToFused(
   narom += rdcast<int>(aromRings.size());
 }
 
-bool isAtomCandForArom(const Atom *at, const ElectronDonorType edon) {
+bool isAtomCandForArom(const Atom *at, const ElectronDonorType edon,
+                       bool allowThirdRow = true, bool allowTripleBonds = true,
+                       bool allowHigherExceptions = true, bool onlyCorN = false,
+                       bool allowExocyclicMultipleBonds = true) {
```

Comment:
I find allowExocyclicMultipleBonds confusing, I assume you mean allowExocyclicNonSingleBonds?

Reply:
yeah, but I think "multiple bond" sounds a lot better than "non-single bond"